### PR TITLE
Darksettings

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/gxr5bh9f77trt987wj8kwdpupm

[download apk](http://filestorage.bluewallet.io:1488/1593565974.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/07jx27q6cdag8xr7fh206bqkjw

[download apk](http://filestorage.bluewallet.io:1488/1593567723.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/gr0wcy2g74z59xngxn82bvx06m

[download apk](http://filestorage.bluewallet.io:1488/1593570469.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/1gtdxm85t4mm2q6a1haud4eaz8

[download apk](http://filestorage.bluewallet.io:1488/1593571256.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/jzbh5dmjxau8ey4yq4g1b83wpw

[download apk](http://filestorage.bluewallet.io:1488/1593607678.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/q76t8hab68eekkd53a5nvyqa68

[download apk](http://filestorage.bluewallet.io:1488/1593640755.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/apr0vrv6nx1gd8um9ah9u07x94

[download apk](http://filestorage.bluewallet.io:1488/1593643907.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/d1mtnjrbe5xxanyrv0k2pzjezw

[download apk](http://filestorage.bluewallet.io:1488/1593688073.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/8f96yktuh99bv84qjrzjv0qf64

[download apk](http://filestorage.bluewallet.io:1488/1593705175.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/0fxhgm67heu71b3x0ert1ev4rr

[download apk](http://filestorage.bluewallet.io:1488/1593776772.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/926vchxyt55wzw8fhewr2j8tkw

[download apk](http://filestorage.bluewallet.io:1488/1593786713.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/2g1pumn7m2amqd7417t0858zcm

[download apk](http://filestorage.bluewallet.io:1488/1593791956.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/tpdezgbae3yn0tctv6pnb04km4

[download apk](http://filestorage.bluewallet.io:1488/1593810934.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/5gc23b629hjtb40y3vyetc5m2r

[download apk](http://filestorage.bluewallet.io:1488/1593831310.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/kc947f9p85ubqazc55b6tb8b9c

[download apk](http://filestorage.bluewallet.io:1488/1593833361.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/wafkk1ufc3j8yxxt0nwa3fkc58

[download apk](http://filestorage.bluewallet.io:1488/1593833858.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/athz39rhh9dg4q9mfphfk8j8tg

[download apk](http://filestorage.bluewallet.io:1488/1593900839.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/w9wccea9yvkpe6pvhb8vmxh51g

[download apk](http://filestorage.bluewallet.io:1488/1593958504.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/nrzeexfc8v9xbh2beunw65rknr

[download apk](http://filestorage.bluewallet.io:1488/1593959835.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/6wf4w1qqb0vhc61mtxhugedy5w

[download apk](http://filestorage.bluewallet.io:1488/1593961685.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/3vc9d60yj7ff8u1k0drpr8mt9r

[download apk](http://filestorage.bluewallet.io:1488/1593963932.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/8bpug0rdmdqekcwm7qhw0dawwr

[download apk](http://filestorage.bluewallet.io:1488/1593970787.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/9trnmmva535jf6mjxv7qpynez4

[download apk](http://filestorage.bluewallet.io:1488/1594008993.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/vkpr3f0yf36xq6pg5yuxw4patm

[download apk](http://filestorage.bluewallet.io:1488/1594038968.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/tep5q366ywtnga88vu7qzz5gn4

[download apk](http://filestorage.bluewallet.io:1488/1594053143.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/848fvwv663t7t5hdxjkapwx9xg

[download apk](http://filestorage.bluewallet.io:1488/1594062597.apk)

**Comment by ncoelho** [comment]
> https://appetize.io/app/848fvwv663t7t5hdxjkapwx9xg

This is crashing, can you check @marcosrdz

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/gparyjywpnat53vwh61u04qj6r

[download apk](http://filestorage.bluewallet.io:1488/1594134618.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/79y9qz05u2zh9zvug6abapq6vc

[download apk](http://filestorage.bluewallet.io:1488/1594139310.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/27e4tuhqpky8uehveqd1ha6fu4

[download apk](http://filestorage.bluewallet.io:1488/1594580540.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/z9zy14uw763e6nmmr2zmuawry4

[download apk](http://filestorage.bluewallet.io:1488/1594656177.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/fm0gghxahv6a354wq5wvpue6mr

[download apk](http://filestorage.bluewallet.io:1488/1594657105.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/tktun6u8x5cbf4uf9qx36thfng

[download apk](http://filestorage.bluewallet.io:1488/1594662597.apk)

File: components/themes.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    success: '#202020',
+    successCheck: '#0A84FF',
+    buttonBlueBackgroundColor: '#202020',
+    scanLabel: 'rgba(255,255,255,.2)',
+    labelText: '#ffffff',
+  },
+};
+
+export class BlueCurrentTheme {}
+BlueCurrentTheme.colors = Appearance.getColorScheme() === 'dark' ? BlueDarkTheme.colors : BlueDefaultTheme.colors;
```
Comment (Overtorment) [inline components/themes.js:103]: weird flex but ok

File: screen/receive/details.js [hunk 2]
```diff
@@ -49,6 +50,89 @@ const ReceiveDetails = () => {
   const [isCustom, setIsCustom] = useState(false);
   const [isCustomModalVisible, setIsCustomModalVisible] = useState(false);
   const { navigate, goBack } = useNavigation();
+  const { colors } = useTheme();
+  const styles = StyleSheet.create({
```
Comment (Overtorment) [inline screen/receive/details.js:54]: styles migrated from bottom of the file to the top when we agreed to keep them at the bottom for better readability when you open component file in your IDE. this is not  deal breaker but now Im trying to put my styles at the bottom always

**Pull request merged**